lib/ansible/module_utils/k8s/common.py

```diff
@@ -139,46 +139,37 @@ def get_api_client(self, **auth_params):
         auth_params = auth_params or getattr(self, 'params', {})
         auth = copy.deepcopy(auth_params)
 
-        configuration = kubernetes.client.Configuration()
+        # Pull authorization settings from environment variables
```

Comment:
Comment here should be more extensive and state precisely what order we're processing configuration.